src/styles/split.scss

```diff
@@ -0,0 +1,34 @@
+.split {
+    display: flex;
+  
+    > * {
+        position: relative;
+        display: flex;
+        flex-shrink: 1;
+        flex-grow: 1;
+    }
+    > video {
+        object-fit: cover;
+    }
+}
+/**
+* Vertical splitting
+*/
+.split-vertical {
+flex-direction: row;
+}
+.split-vertical > * {
+min-height: 100%;
+overflow-y: hidden;
+}
+/**
+* Horizontal splitting
+*/
+.split-horizontal {
+flex-direction: column;
+}
+.split-horizontal > * {
+min-width: 100%;
+max-height: 50%;
+overflow-x: hidden;
+}
```

Comment:
the original css was 
```css
.split-video {
  width: 50%;
}
.left-video {
  left: 0;
}
.right-video {
  right: 0;
}
```
I'd need a more detailed explanation of the reasoning behind these classes.
`.split` can also have a detrimental effect on content.